packages/manager/apps/dedicated/client/app/dedicated/server/server.routing.js

```diff
@@ -39,6 +42,17 @@ export default /* @ngInject */ ($stateProvider) => {
           chapter1: 'dedicated',
         }),
       worldPart: /* @ngInject */ (coreConfig) => coreConfig.getRegion(),
+
+      breadcrumb: /* @ngInject */ (server) => server.displayName,
+
+      bringYourOwnImage: /* @ngInject */ ($q, $stateParams, Server) =>
+        Server.getBringYourOwnImage($stateParams.productId).catch((error) => {
+          if (error.status === 404) {
+            return null;
+          }
+
+          return $q.reject(error);
+        }),
```

Comment:
**nitpick**
```suggestion
        Server.getBringYourOwnImage($stateParams.productId).catch((error) => (error.status === 404 ? null : $q.reject(error))),
```

Reply:
Let's Prettier doing this for us;)
Thanks for the comment!